.eslintrc.yml

```diff
@@ -1,3 +1,4 @@
 extends: airbnb-base
 rules:
   global-require: off
+  comma-dangle: 0
```

Comment:
This change to ESLint rules is not noted in the PR overview. Has this change been discussed? (I'm out of the loop.)

Reply:
I didn't mention it because it got removed in an old unapproved PR, but yea it was causing issues https://github.com/fourkitchens/emulsify-gulp/issues/81#issuecomment-411159428